packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { isNullOrUndefined } from 'util';
+import * as vscode from 'vscode';
+import { hasRootWorkspace, OrgAuthInfo } from '../util';
+import { BrowserNode, NodeType } from './index';
+
+export class MetadataOutlineProvider
+  implements vscode.TreeDataProvider<BrowserNode> {
+  private defaultOrg: string | undefined;
+
+  private _onDidChangeTreeData: vscode.EventEmitter<
```

Comment:
We don't use the underscore prefix for private methods, setting it as private is enough.

Reply:
There were a few places across our repo where I'd seen the underscore prefix be used specifically for this tree view method, but I can remove it. I'll change the name to internalOnDidChangeTreeData or something of the sort so it doesn't conflict with the the public onDidChangeTreeData method.